spec/support/features/idv_step_helper.rb

```diff
@@ -78,4 +85,18 @@ def complete_idv_steps_with_gpo_before_confirmation_step(user = user_with_2fa)
   def complete_idv_steps_before_step(step, user = user_with_2fa)
     send("complete_idv_steps_before_#{step}_step", user)
   end
+
+  private
```

Comment:
I'm going to flip this back to draft until #6448 is merged, since we can avoid these helper methods using the constants introduced there instead.